packages/loot-design/src/components/budget/rollover/BudgetSummary.js

```diff
@@ -410,4 +423,8 @@ export default React.memo(function BudgetSummary({ month }) {
       </NamespaceContext.Provider>
     </View>
   );
-});
+}
+
+export default withRouter(
+  connect(state => ({ localPrefs: state.prefs.local }), actions)(BudgetSummary)
+);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏do you actually need router in this component?

```suggestion
export default connect(state => ({ localPrefs: state.prefs.local }), actions)(BudgetSummary);
```

Reply:
I've made that change and everything still works properly.  Thank.